api/models/service.go

```diff
@@ -35,7 +27,7 @@ func ListServices() (Services, error) {
 	for _, stack := range res.Stacks {
 		tags := stackTags(stack)
 
-		if tags["System"] == "convox" && tags["Type"] == "service" {
+		if tags["System"] == os.Getenv("RACK") && tags["Type"] == "service" {
```

Comment:
Nice fix. Though this might cause trouble on some non-standard racks. Like demo...